httpclient/logging_round_tripper.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright © 2024 Acronis International GmbH.
+
+Released under MIT license.
+*/
+
+package httpclient
+
+import (
+	"context"
+	"fmt"
+	"github.com/acronis/go-appkit/httpserver/middleware"
+	"github.com/acronis/go-appkit/log"
+	"net/http"
+	"time"
+)
+
+// LoggerMode represents a mode of logging.
+type LoggerMode string
```

Comment:
What do you think about renaming `LoggerMode` to `LoggingMode`? It better reflects that this is the mode of logging, not the logger itself. Additionally, it would provide more consistent naming across related types: `LoggingMode`, `LoggingRoundTripper`, and `LoggingRoundTripperOpts`.